migrations/20160806_add_missing_webhook_type.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+/****************************************
+ * Author: Blade Barringer @crookedneighbor
+ *
+ * Reason: Webhooks now support a type
+ * property. Old webhooks are of type
+ * "taskActivity", so this migration updates
+ * the data to have the correct type.
+ ***************************************/
+
+global.Promise = require('bluebird');
+const TaskQueue = require('cwait').TaskQueue;
+const logger = require('./utils/logger');
+const Timer = require('./utils/timer');
+const connectToDb = require('./utils/connect').connectToDb;
+const closeDb = require('./utils/connect').closeDb;
+const validator = require('validator');
+
+const timer = new Timer();
+
+// const DB_URI = 'mongodb://username:password@dsXXXXXX-a0.mlab.com:XXXXX,dsXXXXXX-a1.mlab.com:XXXXX/habitica?replicaSet=rs-dsXXXXXX';
+const DB_URI = 'mongodb://localhost/prod-copy-4';
+
+const LOGGEDIN_DATE_RANGE = {
+  $gte: new Date("2014-01-01T00:00:00.000Z"),
+  $lte: new Date("2014-06-01T00:00:00.000Z"),
+};
+
+let Users;
+
+connectToDb(DB_URI).then((db) => {
+  Users = db.collection('users');
+
+  return Promise.resolve();
+})
+.then(findUsersWithWebhooks)
+.then(correctWebhooks)
+.then(() => {
+  timer.stop();
+  closeDb();
+}).catch(reportError);
+
+function reportError (err) {
+  logger.error('Uh oh, an error occurred');
+  closeDb();
+  timer.stop();
+  throw err;
```

Comment:
since it's called by `catch` I think the error will get swallowed because you could recover it with another catch statement. The workaround is to use setTimeout inside the catch